content/en/docs/started/workstation/minikf-aws.md

```diff
@@ -0,0 +1,185 @@
++++
+title = "Deploy Kubeflow using MiniKF on AWS"
+description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via AWS"
+weight = 40
++++
+
+![MiniKF latest
+version](https://www.arrikto.com/wp-content/uploads/2020/11/minikf-latest-version-aws.svg
+"MiniKF latest version")
+
+This guide describes how to launch a MiniKF instance on AWS. MiniKF is a single
+VM solution on the AWS Marketplace, and installs:
+
+- Kubernetes (using [Minikube](https://minikube.sigs.k8s.io/docs/))
+- Kubeflow
+- [Kale](https://github.com/kubeflow-kale/kale), an orchestration and workflow
+  tool for Kubeflow that enables you to run complete data science workflows
+  starting from a notebook
+- Arrikto's [Rok](https://www.arrikto.com/rok-data-management/), a data
+  management software for data versioning and reproducibility
+
+### Installing MiniKF on AWS
+
+To install MiniKF on AWS, follow the steps below:
+
+1. Go to the [MiniKF page](https://aws.amazon.com/marketplace/pp/B08MBGH311) on
+   the AWS Marketplace.
+1. Click on the **Continue to Subscribe** button.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-page.png"
+       alt="Continue to Subscribe"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Click on the **Accept Terms** button.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-accept-terms.png"
+       alt="Accept Terms"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Wait for a few minutes and the click on **Continue to Configuration**.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-continue-configuration.png"
+       alt="Continue to Configuration"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Select a **Region**. We recommend you keep the default values for **Delivery
```

Comment:
The Google and Msft (?) style guidelines suggest using "you" (second person) - to talk to the users directly. Let's try this, though it uses passive voice instead of the active one. 

```suggestion
1. Select a **Region**. You're recommended to keep the default values for **Delivery
```